CodeReview/src/overlays/CodeReviewCommentOverlay.cpp

```diff
@@ -0,0 +1,72 @@
+/***********************************************************************************************************************
+**
+** Copyright (c) 2015 ETH Zurich
+** All rights reserved.
+**
+** Redistribution and use in source and binary forms, with or without modification, are permitted provided that the
+** following conditions are met:
+**
+**    * Redistributions of source code must retain the above copyright notice, this list of conditions and the following
+**      disclaimer.
+**    * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the
+**      following disclaimer in the documentation and/or other materials provided with the distribution.
+**    * Neither the name of the ETH Zurich nor the names of its contributors may be used to endorse or promote products
+**      derived from this software without specific prior written permission.
+**
+**
+** THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES,
+** INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+** DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+** SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+** SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+** WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+** OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+**
+***********************************************************************************************************************/
+
+#include "CodeReviewCommentOverlay.h"
+
+#include "VisualizationBase/src/declarative/GridLayoutFormElement.h"
+
+#include "VisualizationBase/src/declarative/DeclarativeItem.hpp"
+
+#include "VisualizationBase/src/items/Item.h"
+
+#include "VisualizationBase/src/VisualizationManager.h"
+
+namespace CodeReview
+{
+
+DEFINE_ITEM_COMMON(CodeReviewCommentOverlay, "item")
+
+CodeReviewCommentOverlay::CodeReviewCommentOverlay(Visualization::Item* associatedItem, const StyleType* style) :
+	Super{{associatedItem}, style}
+{
+	setAcceptedMouseButtons(Qt::AllButtons);
+}
+
+void CodeReviewCommentOverlay::determineChildren()
+{
+	Super::determineChildren();
+
+	qreal factor = Visualization::VisualizationManager::instance().mainScene()->mainViewScalingFactor();
```

Comment:
Hmm, instead of doing this, try setting the `ItemIgnoresTransformations` flag and see if that works for you.